hub/powertoys/file-explorer.md

```diff
@@ -23,27 +23,38 @@ File Explorer add-ons currently include:
   - G-code files (.gcode)
   - STL files (.stl)
 
-## Preview Pane
 
-Preview Pane is an existing feature in the Windows File Explorer which shows a lightweight, rich, read-only preview of the file's contents in the view's reading pane. PowerToys adds four extensions: Markdown, SVG, PDF, and G-code. PowerToys also adds support for source code files (for more than 150 file extensions).
+## Thumbnail preview
+
+Showing thumbnails is a built-in Windows feature. For thumbnail preview, PowerToys adds multiple extensions: SVG, PDF, G-code and STL.
+
+### Enabling PowerToys Thumbnail previews
+
+To enable, set which you would like enabled to **On**.
+
+## Preview Pane previewers in PowerToys
 
-### Enabling Preview Pane
+Preview Pane is an existing feature in the Windows File Explorer which allows you to see a preview of the file's contents in the view's reading pane. PowerToys adds multiple extensions: Markdown, SVG, PDF, and G-code. In addition to those, PowerToys also adds support for source code files (for more than 150 file extensions).
 
-To enable, first set all to **On**.
+### Enabling PowerToys Preview Pane previewers
+
+To enable, set which you would like enabled to **On**.
 
 ![PowerToys Settings Enable File Explorer screenshot.](../images/powertoys-settings-fileexplorer.png)
 
+### Enabling Preview Pane in File Explorer on Windows
+
 > [!NOTE]
-> Windows Explorer has an additional setting that needs to be checked in order for preview handlers to work. Open Explorer's Folder options, go to tab "View", under "Advanced settings" check **Show preview handlers in preview pane**.
+> There is an advanced setting in Windows that may be unchecked. If previewers don't seem to work, including ones like jpeg images, please verify this setting is checked. Open Explorer's Folder options, go to tab "View", then under "Advanced settings" check **Show preview handlers in preview pane**.
```

Comment:
```suggestion
> There is an advanced setting in Windows that might be turned off. If preview handlers (including ones like jpeg images) don't seem to work, please verify this setting is turned on. Open Explorer's Folder options, go to tab "View", then under "Advanced settings" check **Show preview handlers in preview pane**.
```

I've seen this "may versus might" in other docs actively being changed. ("may" implies permission, "might" implies chance)
A _setting_ is not really _unchecked_ - at least not in the way you mean it (a UI control would be)
Also: "previewers" sounds like you created a new word.